Tempura.podspec

```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
   s.source           = { :git => 'https://github.com/BendingSpoons/tempura-swift.git', :tag => s.version.to_s }
 
   s.ios.deployment_target = '11.0'
-  s.dependency 'Katana', '>= 5.0', '< 6'
+  s.dependency 'Katana', '>= 5.0', '< 7'
```

Comment:
I don't think we'll be able to support both version 5 and version 6, given that the signature of `StoreListener` changed